composer.json

```diff
@@ -23,14 +23,14 @@
         "issues": "https://github.com/oscarotero/dispatcher/issues"
     },
     "require": {
-        "php": "^7.0|^8.0",
+        "php": "7.1 - 8.4",
```

Comment:
Please don't remove 7.0 compatibility.
And don't limit the max supported version to 8.4 (because it won't be compatible with 8.5). 
I think the previous state is still correct, no need to change it